### PR TITLE
Add dungeon score title and sound

### PR DESCRIPTION
**Pull request by kevinthegreat1**

### CODE REVIEW

File: src/main/java/de/hysky/skyblocker/skyblock/dungeon/DungeonScore.java [hunk 1]
```diff
@@ -36,12 +39,30 @@ public static void tick() {
...
             }
-            if (CONFIG.enableDungeonScore300 && !sent300 && score >= 300) {
-                MessageScheduler.INSTANCE.sendMessageAfterCooldown(Constants.PREFIX.get().getString() + CONFIG.dungeonScore300Message.replaceAll("\\[score]", "300"));
+            if (!sent300 && score >= 300) {
+                if (CONFIG.enableDungeonScore300Message) {
+                    MessageScheduler.INSTANCE.sendMessageAfterCooldown(Constants.PREFIX.get().getString() + CONFIG.dungeonScore300Message.replaceAll("\\[score]", "300"));
+                }
+                if (CONFIG.enableDungeonScore300Title) {
+                    client.inGameHud.setDefaultTitleFade();
+                    client.inGameHud.setTitle(Constants.PREFIX.get().append(CONFIG.dungeonScore300Message.replaceAll("\\[score]", "300")));
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/skyblock/dungeon/DungeonScore.java:61]: Is using the player's score message appropriate here, the config seems to state it being used only for chat messages. Longer score messages could also easily go off screen.

Comment (kevinthegreat1) [reply src/main/java/de/hysky/skyblocker/skyblock/dungeon/DungeonScore.java:61]: I think it's fine. I don't want to clutter the config.

**Comment by Thsgun** [comment]
how do i test this

**Review by AzureAaron — APPROVED** [review]
The design of this should probably be reevaluated later, not so friendly with high GUI scales.

**Pull request merged**